packages/app-layout/src/AppLayoutContent.tsx

```diff
@@ -0,0 +1,112 @@
+import { Fragment, PropsWithChildren } from 'react';
+import { Prose } from '@ag.ds-next/react/prose';
+import { PageContent } from '@ag.ds-next/react/content';
+import { Callout } from '@ag.ds-next/react/callout';
+import { CoreProvider } from '@ag.ds-next/react/core';
+import { Text } from '@ag.ds-next/react/text';
+import { TextLink } from '@ag.ds-next/react/text-link';
+import { DirectionLink } from '@ag.ds-next/react/direction-link';
+import { hrefs } from './utils';
+import { Stack } from '@ag.ds-next/react/stack';
+import { Divider } from '@ag.ds-next/react/divider';
+
+export type ExpectedClaims = {
+	name?: string;
+	given_name?: string;
+	family_name?: string;
+
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	[others: string]: any;
+};
+
+const isEmptyString = (t: string | undefined) => (t?.trim() ?? '') === '';
+
+// This exists as its own package in ag.common.
+// Likely cleaner to re-use it here but makes consuming it more error prone (peer deps).
+const HelpCallout = () => {
+	return (
+		<CoreProvider>
+			<Callout title="Need more help?">
+				<Text>
+					Email{' '}
+					<TextLink href="mailto:exportservice@aff.gov.au">
+						exportservice@aff.gov.au
+					</TextLink>
+				</Text>
+				<Text>
+					Call <TextLink href="tel:1800571125">1800&nbsp;571&nbsp;125</TextLink>
+					, Monday to Friday, 9 am to 5 pm AEST
+				</Text>
+			</Callout>
+		</CoreProvider>
+	);
+};
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Unnecessary? Isn't this used on L78?

Reply:
props are not used but i want the type signature preserved

Reply:
Ah, I see